css/index.css

```diff
@@ -104,14 +136,45 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
+/*    background: black;*/
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
```

Comment:
instead of using nth-child you could make 2 classes all the boxes could have .box and then each could have a numbered class, example .box .box1, .box .box2  and so on